AutoDarkModeApp/ViewModels/WallpaperPickerViewModel.cs

```diff
@@ -212,15 +227,29 @@ private void LoadSettings()
         }
         else if (currentType == WallpaperType.Spotlight)
         {
-            //TODO: Need a better method
-            //GlobalWallpaperSource = new BitmapImage(new Uri(Environment.GetFolderPath(Environment.SpecialFolder.ApplicationData) + @"\Microsoft\Windows\Themes\TranscodedWallpaper"));
-            GlobalWallpaperSource = null;
+            //TODO: It is necessary to determine the correct position of focusing wallpaper
+            if (Environment.OSVersion.Version.Build >= (int)WindowsBuilds.Win11_24H2)
+            {
+                GlobalWallpaperPath = @"C:\Windows\SystemApps\MicrosoftWindows.Client.CBS_cw5n1h2txyewy\DesktopSpotlight\Assets\Images\image_1.jpg";
```

Comment:
Most people will have Windows installed in `C:`, but there may be cases that it's not.